eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -43,6 +43,18 @@ stages:
         dependsOn: Signing
         condition: and(succeeded(), ne(variables['SetDevVersion'], 'true'), ne(variables['Skip.Release'], 'true'), ne(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net-pr'))
         jobs:
+          - job: InstallationCheck
```

Comment:
Any reason this is a separate job?

Reply:
I think the installation check should be done before the other jobs and this just doesn't quite fit into the definition of "CreateReleaseTag." I thought about putting this as steps within PublishPackage. Do you think this approach would be better?

Reply:
I would like it to fail as early in the process as possible. I would probably put it with the tagging job as we do some other checks there as well. Doing just this in a separate job causes us to use another agent and also causes more overhead that probably isn't necessary just for the one check.